services/opm/opm-version.service.js

```diff
@@ -0,0 +1,77 @@
+'use strict'
+
+const { renderVersionBadge } = require('../version')
+const { BaseService, NotFound, InvalidResponse } = require('..')
+
+module.exports = class OpmVersion extends BaseService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'opm/v',
+      pattern: ':user/:moduleName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'OPM',
+        namedParams: {
+          user: 'openresty',
+          moduleName: 'lua-resty-lrucache',
+        },
+        staticPreview: this.render({ version: 'v0.08' }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return {
+      label: 'opm',
+    }
+  }
+
+  static render({ version }) {
+    const { label: defaultLabel } = this.defaultBadgeData
+    return renderVersionBadge({
+      label: defaultLabel,
+      version,
+    })
+  }
```

Comment:
We can remove this function altogether and just leverage the `renderVersionBadge` helper everwhere (details inline elsewhere on how)